src/middlewared/middlewared/etc_files/pwd_db.py

```diff
@@ -2,4 +2,5 @@
 
 
 async def render(service, middleware):
+    middleware.logger.debug("XXXX::: GENERATING MASTERPW")
```

Comment:
Why this

Reply:
Whoops. Didn't realize the branch wasn't clean.